nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryRecord.java

```diff
@@ -480,6 +491,18 @@ public int getRecordsRead() {
         };
     }
 
+    private SchemaPlus createRootSchema(final CalciteConnection calciteConnection) {
+        final SchemaPlus rootSchema = calciteConnection.getRootSchema();
+        rootSchema.add("RPATH", ScalarFunctionImpl.create(ObjectRecordPath.class, "eval"));
```

Comment:
Is there a way to register this with a case-insensitive name? If I do a `select * from flowfile where rpath(x,y)` it says it can't find a function `rpath` with the right signature. However if I do `select * from flowfile where RPATH(x,y)` it works fine.

Reply:
I don't see a way.